### DISCUSSION TITLE
Pluggability

### DISCUSSION THREAD

**Discussion by lsap** [Bogdan17-bit/Trading-Platform#8] [Ideas]
Your codebase is growing. Do you like to split the whole thing into core and plugins (say, logging)?

**Comment by Bohdan17-bit** [comment]
I'm thinking about codebase improvement, there are some problems with memory leak and "quick solutions" in the project. 
I will think about it, because the project does not have a logging system yet.

**Comment by Bohdan17-bit** [comment]
Isap, can you give me your email?

**Reply by lsap** [reply]
DMed
